chemprop/cli/common.py

```diff
@@ -75,7 +75,8 @@ def add_common_args(parser: ArgumentParser) -> ArgumentParser:
 
 - ``V1``: Corresponds to the original configuration employed in the Chemprop V1
 - ``V2``: Tailored for a broad range of molecules, this configuration encompasses all elements in the first four rows of the periodic table, along with iodine. It is the default in Chemprop V2.
-- ``ORGANIC``: This configuration is designed specifically for use with organic molecules for drug research and development and includes a subset of elements most common in organic chemistry, including H, B, C, N, O, F, Si, P, S, Cl, Br, and I.""",
+- ``ORGANIC``: This configuration is designed specifically for use with organic molecules for drug research and development and includes a subset of elements most common in organic chemistry, including H, B, C, N, O, F, Si, P, S, Cl, Br, and I.
+- ``RIGR``: This configuration is designed to featurize the molecular graph using only the resonance-invariant atom and bond features which are included in the default atom and bond featurizers.""",
```

Comment:
```suggestion
- ``RIGR``: Modified V2 (default) featurizer using only the resonance-invariant atom and bond features.""",
```

"This configuration is designed to featurize the molecular graph [...]"

All of these do that - don't need to repeat it

I also explicitly gave the name of the default featurizer (save people having to read which is default) to make it more clear that RIGR 'inherits' from V2 and is also appropriate for many species.